locale-utils.js

```diff
@@ -0,0 +1,74 @@
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+
+// node.js needs Intl.PluralRules polyfill
+require("intl-pluralrules");
```

Comment:
![its-working-its-working](https://user-images.githubusercontent.com/71928/46970288-4f518780-d07e-11e8-9721-c4ed5fbb3606.gif)

Reply:
Nice! Happy to hear that was it!